### ISSUE TITLE
Instant mode turns off interval mode

### ISSUE THREAD

**Issue by macjeff** [simplextech/udi-poly-linktap#9]
First I have a feeling we already discussed this but I cant remember for sure.

Anyways when I use the app in linktap to turn on a device that has an Interval mode it asks if I want to leave the interval mode program running and I say yes (leave it checked is yes and unchecked is no).  So like if I water every day from 9-11 am and I run instant mode at 3pm it will let the instant mode run and when completed re-activate the interval program.

Yesterday I used the ISY to run a program to turn on the valve for 30 min.  It worked fine and turned off in 30 min.  But today my interval mode did not run so I went in and sure enough the program had been disabled.  So I re-enabled it.

Can you have this option for the NodeServer?  I cant see why anyone would want to disable it running it once so at least the default should be to re-activate.

thanks!!!

**Comment by simplextech** [comment]
` autoBack: Optional. String or Boolean type. "true" for automatically re-activating the previous watering plan after watering in Instant Mode is completed. `

The above option is already included in the instant on api call.  It's in the activate_instan_on function.  I'm wondering if the api works or if there's something missing from the documentation.

I've made a slight change to clean it up a bit but we'll see if it makes any difference.

**Comment by simplextech** [comment]
BTW - update was pushed.

**Comment by macjeff** [comment]
I sent an ISY program for 1 minute so as soon as it posts I will run the program and see what happens at the end.

**Comment by macjeff** [comment]
I just tested and it worked.   BUT I figured out what the issue is.  I forgot yesterday I issued a STOP command before finished.
But then I tested and the app works the same way which is stupid.   
Is there any way for you to issue the same command to keep it going if a stop is entered.
But there is a workaround I found....(it was there the whole time  LOL)
My program is instant off command.  the stop command is what killed the interval mode.  But I did find I can enter one extra line.  Set Valve to interval mode and that will turn on and go back to the last set program.
